pom.xml

```diff
@@ -49,6 +49,10 @@
         <skipTests>false</skipTests>
         <skipUnitTests>${skipTests}</skipUnitTests>
         <skipIntegrationTests>${skipTests}</skipIntegrationTests>
+
```

Comment:
There are 3 POM files in the project:
- main (Apache)
- Confluent
- preformance tests

Not sure about the last one, but for sure you need to sync changes between Apache and Confluent

Reply:
thanks, didn't know that